lms/djangoapps/discussion_api/tests/test_serializers.py

```diff
@@ -586,6 +587,7 @@ def test_update_all(self):
                 "closed": ["False"],
                 "pinned": ["False"],
                 "user_id": [str(self.user.id)],
+                "read": ["False"],
```

Comment:
Added `ddt` for both `True` and `False` values of `read` in `test_update_all` case